hack/generate/gen-test-framework.sh

```diff
@@ -13,8 +13,8 @@ cd test/test-framework
 
 # Ensure test-framework is up-to-date with current Go project dependencies.
 # NOTE: the SDK dependency version still needs updating on a new release.
+echo "$(../../build/operator-sdk print-deps)" > go.mod
```

Comment:
Any reason for this change? The reason I had `sdk_version=...` above this command is we want to save the current version and add it back to `go.mod` later. Otherwise `version` in `require github.com/operator-framework/operator-sdk ${version}` will be updated every time a new commit is made in the master branch.

Reply:
On the commit prior to this change, the `gen-test-framework.sh` script is failing for me. I get this in the `test/test-framework` dir:

```
$ go list -m -f '{{.Version}}' github.com/operator-framework/operator-sdk
go: github.com/operator-framework/operator-sdk@v0.12.1-0.20191114194531-79e6369540c6 requires
	k8s.io/kubectl@v0.0.0: reading k8s.io/kubectl/go.mod at revision v0.0.0: unknown revision v0.0.0
```

But you're right. If this gets committed, it seems like every PR will have to regenerate the test framework, which would be a non-starter.

The above error happens because we use `replace` to pull in `operator-sdk` at a different version than what the existing test-framework's `go.mod` file is compatible with. More specifically, the SDK's `go.mod` file that has been updated for k8s-1.16 has a `require` line for `k8s.io/kubectl v0.0.0` and a `replace` line to fill in the correct version. But since `replace` lines are only honored in the main module (not dependent modules), and the test-framework doesn't have a `replace` for `k8s.io/kubectl`, we have a problem.

To fix this, I think we can just hardcode the operator-sdk `require` line to `v0.0.0` since we always use a `replace` line.